examples/eks/addon.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: Addon
+metadata:
+  name: sample-addon
+  labels:
+    example: "true"
+spec:
+  forProvider:
+    region: eu-central-1
```

Comment:
please change to `us-east-1` that it fits to the clusterRef

Reply:
Fixed